Makefile

```diff
@@ -67,7 +67,7 @@ fmt:
 lint:
 	$(GOVET) $(PACKAGES)
 	@cat /dev/null > $(LINT_LOG)
-	@$(foreach pkg, $(PACKAGES), $(GOLINT) $(pkg) | grep -v -e thrift-gen -e thrift-0.9.2 >> $(LINT_LOG) || true;)
+	@$(foreach pkg, $(PACKAGES), $(GOLINT) $(pkg) | grep -v -e IndexService.go -e thrift-gen -e thrift-0.9.2 >> $(LINT_LOG) || true;)
```

Comment:
Should be more specific, like `mocks/IndexService.go`.

Alternatively, we could exclude all mocks dirs (`/mocks/`), not much value in linting those.